packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py

```diff
@@ -563,36 +608,194 @@ def clear_job_annotations(job_id: int) -> None:
+        )
+
+        try:
+            response = uploader.upload_file(
+                url,
+                dataset_path,
+                query_params={"format": format_name, "filename": dataset_path.name},
+                meta={"filename": dataset_path.name},
+            )
+        except Exception as ex:
+            logger.exception(f"Exception occurred while importing GT annotations: {ex}\n")
+            raise
+
+        request_id = json.loads(response.data).get("rq_id")
+        assert request_id, "CVAT server have not returned rq_id in the response."
+
+        while True:
+            try:
+                (request_details, _) = client.api_client.requests_api.retrieve(request_id)
+            except exceptions.ApiException as ex:
+                logger.exception(f"Exception occurred while importing GT annotations: {ex}\n")
+                raise
+
+            if (
+                request_details.status.value
+                == models.RequestStatus.allowed_values[("value",)]["FINISHED"]
+            ):
+                break
+
+            if (
+                request_details.status.value
+                == models.RequestStatus.allowed_values[("value",)]["FAILED"]
+            ):
+                raise Exception(
+                    "Annotations upload failed. "
+                    f"Previous status was: {request_details.status.value}."
+                )
+
+            if timeout is not None and timedelta(seconds=timeout) < (utcnow() - start_time):
+                raise Exception(
+                    "Failed to upload the GT annotations to CVAT within the timeout interval. "
+                    f"Previous status was: {request_details.status.value}. "
+                    f"Timeout: {timeout} seconds."
+                )
+
+            sleep(sleep_interval)
+
+    logger.info(f"GT annotations for the job {job_id} have been uploaded to CVAT.")
+
+
+def get_quality_control_settings(task_id: int) -> models.QualitySettings:
+    logger = logging.getLogger("app")
+
+    with get_api_client() as api_client:
+        try:
+            paginated_data, _ = api_client.quality_api.list_settings(task_id=task_id)
+            assert len(paginated_data["results"]) == 1, (
+                f'CVAT returned {len(paginated_data["results"])}'
+                "quality control settings associated with the task"
+            )
+            return paginated_data["results"][0]
+
+        except (exceptions.ApiException, AssertionError) as e:
+            logger.exception(f"Exception when calling QualityApi.list_settings(): {e}\n")
+            raise
+
+
+def update_quality_control_settings(
+    settings_id: int,
+    *,
+    max_validations_per_job: int = Config.cvat_config.cvat_max_validation_checks,
+    target_metric: str = "accuracy",
+    target_metric_threshold: float = Config.cvat_config.cvat_target_metric_threshold,
+    low_overlap_threshold: float = Config.cvat_config.cvat_low_overlap_threshold,
```

Comment:
This value can be set to the same value as iou threshold or to 1, it only affects warnings